common/scala/src/main/scala/whisk/core/containerpool/Container.scala

```diff
@@ -66,15 +74,3 @@ case class BlackboxStartupError(msg: String) extends ContainerStartupError(msg)
 
 /** Indicates an error while initializing a container */
 case class InitializationError(interval: Interval, response: ActivationResponse) extends Exception(response.toString)
-
-case class Interval(start: Instant, end: Instant) {
-    def duration = Duration.create(end.toEpochMilli() - start.toEpochMilli(), MILLISECONDS)
-}
-
-object Interval {
-    /** An interval starting now with zero duration. */
-    def zero = {
-        val now = Instant.now
-        Interval(now, now)
-    }
-}
```

Comment:
I'm suggesting that container management can be handled outside of an invoker, therefore Container related interfaces do not belong there.